docs/config.rst

```diff
@@ -46,7 +46,12 @@ core
 
     If ``true``, Gordon will continue running even if installed & configured plugins can not be loaded. Plugin exceptions will be logged as warnings with tracebacks.
 
-    If ``false``, Gordon will exit out if it can't load one or more plugins.
+    If ``false``, Gordon will exit out if it can't load one or more plugins
+
+.. option:: metrics='metrics-provider-name'
+
+    The metrics provider to use. Depending on the provider, more
+    configuration may be needed. See provider implementation for details.
```

Comment:
No need for line breaks in actual `rst` files (unlike docstrings).

Reply:
I still think it's easier to read this way. It doesn't mess up the rendering, right?

Reply:
I have word wrapping in my editor, so I think it's personal preference. But if we go this route, then we should adjust the rest of the docs (which is more prevalent in `gordon-gcp` and a bit in `gordon-janitor`) to be consistent.